### PR TITLE
readme: restructure the README.md

### PR DESCRIPTION
**Pull request by shizn**
### What problem does this PR solve?

Update the readme file.

Issue Number: close #xxx

Problem Summary:

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [x] No need to test
  > - [x] I checked and no code files have been changed.
  > 

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Review by breezewish — APPROVED** [review]
Looks very good!

File: README.md [hunk 1]
```diff
@@ -1,123 +1,116 @@
...
-    &nbsp;
-    <a href="https://slack.tidb.io/invite?team=tidb-community&channel=everyone&ref=pingcap-tidb"><img src="https://img.shields.io/badge/-Slack-red?style=social&logo=slack" height=25></a>
-    &nbsp;
-    <a href="https://stackoverflow.com/questions/tagged/tidb"><img src="https://img.shields.io/badge/-Stack Overflow-red?style=social&logo=stackoverflow" height=25></a>
-</div>
+- **[Distributed Transactions](https://www.pingcap.com/blog/distributed-transactions-tidb?utm_source=github&utm_medium=tidb)**: TiDB uses a two-phase commit protocol to ensure ACID compliance, providing strong consistency. Transactions span multiple nodes, and TiDB's distributed nature ensures data correctness even in the presence of network partitions or node failures.
 
-<br />
+- **[Horizontal and Vertical Scalability](https://docs.pingcap.com/tidb/stable/scale-tidb-using-tiup?utm_source=github&utm_medium=tidb)**: TiDB can be scaled horizontally by adding more nodes or vertically by increasing resources of existing nodes, all without downtime. TiDB's architecture separates computing from storage, enabling you to adjust both independently as needed for flexibility and  growth.
```
Comment (sunxiaoguang) [inline README.md:32]:
```suggestion
- **[Horizontal and Vertical Scalability](https://docs.pingcap.com/tidb/stable/scale-tidb-using-tiup?utm_source=github&utm_medium=tidb)**: TiDB can be scaled horizontally by adding more nodes or vertically by increasing resources of existing nodes, all without downtime. TiDB's architecture separates computing from storage, enabling you to adjust both independently as needed for flexibility and growth.
```

File: README.md [hunk 2]
```diff
@@ -1,123 +1,116 @@
...
 
-See the [TiDB Cloud Quick Start Guide](https://docs.pingcap.com/tidbcloud/tidb-cloud-quickstart).
+2. Learn About TiDB SQL: To explore the SQL capabilities of TiDB, refer to the [TiDB SQL documentation](https://docs.pingcap.com/tidb/stable/sql-statement-overview?utm_source=github&utm_medium=tidb).
 
-### Start with TiDB
+3. Use MySQL Driver or ORM to [Build an App with TiDB with TiDB](https://docs.pingcap.com/tidbcloud/dev-guide-overview?utm_source=github&utm_medium=tidb).
 
-See [TiDB Quick Start Guide](https://docs.pingcap.com/tidb/stable/quick-start-with-tidb).
+4. Explore key features, such as [data migration](https://docs.pingcap.com/tidbcloud/tidb-cloud-migration-overview?utm_source=github&utm_medium=tidb), [changefeed](https://docs.pingcap.com/tidbcloud/changefeed-overview?utm_source=github&utm_medium=tidb), [vector search](https://docs.pingcap.com/tidbcloud/vector-search-overview?utm_source=github&utm_medium=tidb), [HTAP](https://docs.pingcap.com/tidbcloud/vector-search-overview?utm_source=github&utm_medium=tidb), [disaster recovery](https://docs.pingcap.com/tidb/stable/dr-solution-introduction?utm_source=github&utm_medium=tidb), etc.
```
Comment (sunxiaoguang) [inline README.md:61]:
```suggestion
4. Explore key features, such as [data migration](https://docs.pingcap.com/tidbcloud/tidb-cloud-migration-overview?utm_source=github&utm_medium=tidb), [changefeed](https://docs.pingcap.com/tidbcloud/changefeed-overview?utm_source=github&utm_medium=tidb), [vector search](https://docs.pingcap.com/tidbcloud/vector-search-overview?utm_source=github&utm_medium=tidb), [HTAP](https://docs.pingcap.com/tidbcloud/tidb-cloud-htap-quickstart?utm_source=github&utm_medium=tidb), [disaster recovery](https://docs.pingcap.com/tidb/stable/dr-solution-introduction?utm_source=github&utm_medium=tidb), etc.
```

File: README.md [hunk 3]
```diff
@@ -1,123 +1,116 @@
...
 
-## Case studies
+- The [contribution map](https://github.com/pingcap/tidb-map/blob/master/maps/contribution-map.md#a-map-that-guides-what-and-how-contributors-can-contribute) lists everything you can contribute.
 
-- [Case studies in English](https://www.pingcap.com/customers/)
-- [中文用户案例](https://cn.pingcap.com/case/)
+- The [community repository](https://github.com/pingcap/community) contains everything else you need.
 
-## Architecture
+- Don't forget to claim your contribution sway by filling in and submitting this [form](https://forms.pingcap.com/f/tidb-contribution-swag).
```
Comment (sunxiaoguang) [inline README.md:90]:
```suggestion
- Don't forget to claim your contribution swag by filling in and submitting this [form](https://forms.pingcap.com/f/tidb-contribution-swag).
```

File: README.md [hunk 4]
```diff
@@ -1,123 +1,116 @@
...
 
 ## License
 
 TiDB is under the Apache 2.0 license. See the [LICENSE](./LICENSE) file for details.
 
+## See Also
+
+- [TiDB Online Playgroud](https://play.tidbcloud.com/?utm_source=github&utm_medium=tidb_readme)
+- TiDB Case Studies: [TiDB Customers](https://www.pingcap.com/customers/), [TiDB 事例記事](https://pingcap.co.jp/case-study/), [TiDB 中文用户案例](https://cn.pingcap.com/case/)
+- [TiDB User Documentation](https://docs.pingcap.com/tidb/stable)
```
Comment (sunxiaoguang) [inline README.md:108]:
```suggestion
- [TiDB User Documentation](https://docs.pingcap.com/tidb/stable?utm_source=github&utm_medium=tidb)
```

File: README.md [hunk 5]
```diff
@@ -1,123 +1,116 @@
...
+- [TiDB Design Docs](/docs/design)
+- [TiDB Release Notes](https://docs.pingcap.com/tidb/dev/release-notes)
```
Comment (sunxiaoguang) [inline README.md:110]:
```suggestion
- [TiDB Release Notes](https://docs.pingcap.com/tidb/dev/release-notes?utm_source=github&utm_medium=tidb)
```

File: README.md [hunk 6]
```diff
@@ -1,123 +1,116 @@
...
+- [TiDB Blog](https://www.pingcap.com/blog/)
```
Comment (sunxiaoguang) [inline README.md:111]:
```suggestion
- [TiDB Blog](https://www.pingcap.com/blog/?utm_source=github&utm_medium=tidb)
```

File: README.md [hunk 7]
```diff
@@ -1,123 +1,116 @@
...
+</a>
 
 ## License
 
 TiDB is under the Apache 2.0 license. See the [LICENSE](./LICENSE) file for details.
 
+## See Also
+
+- [TiDB Online Playgroud](https://play.tidbcloud.com/?utm_source=github&utm_medium=tidb_readme)
+- TiDB Case Studies: [TiDB Customers](https://www.pingcap.com/customers/), [TiDB 事例記事](https://pingcap.co.jp/case-study/), [TiDB 中文用户案例](https://cn.pingcap.com/case/)
```
Comment (sunxiaoguang) [inline README.md:107]:
```suggestion
- TiDB Case Studies: [TiDB Customers](https://www.pingcap.com/customers/?utm_source=github&utm_medium=tidb), [TiDB 事例記事](https://pingcap.co.jp/case-study/?utm_source=github&utm_medium=tidb), [TiDB 中文用户案例](https://cn.pingcap.com/case/?utm_source=github&utm_medium=tidb)
```

**Comment by c4pt0r** [comment]
LGTM

**Review by yudongusa — APPROVED** [review]
Approved by Ed per Xin.

**Review by bb7133 — APPROVED** [review]
LGTM

**Pull request merged**